### ISSUE TITLE
Wall Control support?

### ISSUE THREAD

**Issue by rrauenza** [MGX3D/pegstr#1]
Hi!

This is more of an FYI -

I've forked your repo and am working on trying to get wall control hooks supported.  I have a metal peg board and it tears apart the 3d printed pegs.  But my board also supports Wall Control.   I was thinking of making supports that one could drop onto hooks like this:

https://www.wallcontrol.com/standard-slotted-hook-6-pack-7-8in-reach-black/

The board looks like this ..

https://www.wallcontrol.com/32in-x-16in-galvanized-metal-pegboard-tool-board-panel-metallic/

So the spacing is the same horizontally.

I'm pretty new at OpenSCAD so I'm struggling a bit, so if this is something you'd (or anyone else) be interested in helping support, please let me know.

**Comment by MGX3D** [comment]
Wall Control should work already, I've used it for this (the black one with both holes and slots). I didn't have any problem with that, I wonder if it's the printer tolerance or the material? Can you post some pictures with what's happening?

**Comment by rrauenza** [comment]
It mostly works ... I'm just not satisfied with how hard it is to move the items once inserted (and its an excuse to learn some CAD).   

The 3d printed pegs just aren't super strong -- the ones that break tend to be at the sides, I think because there is less support material?  Or maybe that's where I put pressure on them to remove.

It might be tolerances in my printer, but I also want to make sure the items are secure and don't constantly fall out (like all the metal ones do!), so I don't want to just shrink the pegs

I got frustrated with OpenSCAD's requirement of having epsilon's everywhere for cuts, so jumped into Fusion360.  I've posted some designs on thingiverse -- one of the STLs contains the hook / hanger module.

https://www.thingiverse.com/thing:2852340

Feel free to close...
